sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -179,11 +180,11 @@ export class ECertFileHandler extends ESDCFileHandler {
           this.logger.log(`Uploading ${offeringIntensity} content...`);
           await eCertIntegrationService.uploadContent(
             fileContent,
-            fileInfo.filePath,
+            fileInfo.filePath.slice(0, -4),
```

Comment:
Is this code removing the file extension? If it is there are specific libraries to do it, please check: https://nodejs.org/api/path.html#path_path_parse_path

Why do we need to remove the extension here?

Reply:
i also has the same question

Reply:
Yeah, we can do it. Thanks